tensorflow_datasets/image/tiny_imagenet_test.py

```diff
@@ -0,0 +1,28 @@
+"""TODO(tiny_imagenet): Add a description here."""
```

Comment:
Hi!

It's nice to see fellow GCI competitors also contributing to Tensorflow :) For the unit tests, it's recommended that you use the functions in `testing/fake_data_utils.py` to generate some fake example data so that your `tiny_imagenet_test.py` can run on the examples. Feel free to look at a model of how to do this in my previous PR here: https://github.com/tensorflow/datasets/pull/1276/files#diff-0244ac28688e4faa01b53e70fb87c678. Let me know if this helps!

WZHANG

Reply:
Also, you can probably get rid of the TODO's and lines 19-23 if you don't need them

Reply:
Thanks! Would love to talk more about TensorFlow as a fellow participant!

Reply:
Great, awesome work btw! Feel free to email me if you'd like.